cli/integration/go.mod

```diff
@@ -22,6 +22,8 @@ require (
 	oras.land/oras-go/v2 v2.6.0
 )
 
+replace ocm.software/open-component-model/bindings/go/constructor => ../../bindings/go/constructor
```

Comment:
Do you still need this replace? Or should this now be okay since the schema is merged?